helpdesk/settings.py

```diff
@@ -329,6 +337,10 @@
 # Override it in your own Django settings.py
 HELPDESK_ATTACHMENT_DIR_PERMS = int(getattr(settings, 'HELPDESK_ATTACHMENT_DIR_PERMS', "755"), 8)
 
+HELPDESK_VALID_EXTENSIONS = getattr(settings, 'HELPDESK_VALID_EXTENSIONS', ['.txt', '.asc', '.htm', '.html', '.pdf', '.doc', '.docx', '.odt', '.jpg', '.png', '.eml'])
+
+HELPDESK_VALIDATE_ATTACHMENT_TYPES = getattr(settings, 'HELPDESK_VALIDATE_ATTACHMENT_TYPES', True)
+
```

Comment:
Use `VALID_EXTENSIONS` if it exists and fall back to `HELPDESK_VALID_EXTENSIONS` if empty to maintain backwards compatibility.

Failing there will ned to be clear upgrade notes telling users to change the name when they upgrade.